src/pages/_app.js

```diff
@@ -11,4 +11,4 @@ const MyApp = ({ Component, pageProps }) => {
   );
 };
 
-export default MyApp;
\ No newline at end of file
+export default MyApp;
```

Comment:
Added a newline in accordance with the [eol-last](https://eslint.org/docs/rules/eol-last) rule.